src/language-js/html-binding.js

```diff
@@ -19,12 +19,21 @@ function printHtmlBinding(path, options, print) {
     return path.call(
       (functionDeclarationPath) => {
         const { params } = functionDeclarationPath.getValue();
+        const indentOrGroup = params.length > 1 ? indent : group;
         return concat([
           params.length > 1 ? "(" : "",
-          join(
-            concat([",", line]),
-            functionDeclarationPath.map(print, "params")
+          indentOrGroup(
+            concat([
+              params.length > 1 ? softline : "",
+              group(
+                join(
+                  concat([",", line]),
+                  functionDeclarationPath.map(print, "params")
+                )
+              ),
+            ])
           ),
+          params.length > 1 ? softline : "",
           params.length > 1 ? ")" : "",
```

Comment:
There are many `params.length > 1`, how about a `if/else`.

Reply:
Good enough?

```js
      (functionDeclarationPath) => {
        const { params } = functionDeclarationPath.getValue();
        let [blankOrSoftline, indentOrGroup] = ["", group];
        let [openParen, closeParen] = ["", ""];
        if (params.length > 1) {
          // @ts-ignore
          [blankOrSoftline, indentOrGroup] = [softline, indent];
          [openParen, closeParen] = ["(", ")"];
        }
        return concat([
          openParen,
          indentOrGroup(
            concat([
              blankOrSoftline,
              group(
                join(
                  concat([",", line]),
                  functionDeclarationPath.map(print, "params")
                )
              ),
            ])
          ),
          blankOrSoftline,
          closeParen,
        ]);
      },
```

Reply:
Why not just

```
if (params.length > 1) {
return ...
}

return ...
```

Reply:
Fair enough.